api/v1alpha1/halayerset_webhook_test.go

```diff
@@ -22,8 +22,39 @@ import (
 )
 
 var _ = Describe("HALayerSet Validation", func() {
+	Describe("create HALayerSet", func() {
+		Context("add fence agents with unique names", func() {
+			It("should succeed", func() {
+				haNew := createHALayerSetCR()
+				Expect(haNew.ValidateCreate()).NotTo(HaveOccurred())
+			})
+
+		})
+		Context("add same fence agent name twice", func() {
```

Comment:
Since they are nested under different `Describe` I don't think it's an issue, we will have
- `create HALayerSet` add fence agents with unique names
- `updating HALayerSet` add fence agents with unique names
In the same sense that we have many `It` with the "should fail" text - but each is nested under a different context.